### ISSUE TITLE
Different regions

### ISSUE THREAD

**Issue by AcuityAdsIntegrations** [prebid/prebid-server#3195]
Hi, our adapter `AcuityAds` uses the `host` parameter to generate the endpointUrl for sending the request to different regions. As far as we know, you want to get rid of this decision soon. When will you stop supporting the current approach and what solutions can you recommend?

**Comment by bretg** [comment]
A summary of the long thread in issue https://github.com/prebid/prebid-server/issues/2612 : 

- We will not accept **fully** dynamic DNS names. (The AcuityAds adapter doesn't fall into this category. It's `{{.Host}}.admanmedia.com`)
- There seem to be two reasons that adapters do partially dynamic hosts:
    - **different endpoint per region**: We view this reason as unfriendly and sub-optimal. Publishers generally only enter one set of parameters into their CMS for each adunit. It's not practical to think the global publishers will put different hosts in for you for different users. The way this meant to work is that PBS host companies can deploy different config for each region - i.e. when the global load balancing sends a user to US-East, that Prebid Server config can in-turn send it to you us-east endpoint.
    - **different endpoint per account**: this is one we don't fully agree with, but are not going to force adapters to change.  Some bidders architected their fault tolerance using DNS -- they can determine "account 123 is sending us too much traffic" and rewire their DNS to send some elsewhere. We would discourage this architecture, but will not force a change at the open source level. If a Prebid Server Host Company has a problem with one of these bidders due to this behavior though, they may be disabled by that host company.

So in short, we aren't going to force any bidder to change, but for those that use dynamic endpoints for geo-balancing, we strongly suggest you choose a better solution, which is:

1. in your bidder YAML file, comment your auction endpoint to note that you have several different geographic regions and that PBS host companies should contact you before enabling your adapter.
2. make 'host' an optional parameter for your adapter. PBS Host Companies that deploy your config directly to your regional endpoint will not need to burden their publishers with this purely technical concern.
